### PR TITLE
Test UI travis changes

### PR DESCRIPTION
**Pull request by himdel**
Testing https://github.com/ManageIQ/manageiq-ui-classic/pull/6802
(and https://github.com/ManageIQ/manageiq/pull/20018 without which we know it fails)

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,9 @@ matrix:
   fast_finish: true
 env:
   global:
-  - REPOS=
+  - REPOS=manageiq#20018,manageiq-ui-classic#6802
   matrix:
-  - TEST_REPO=manageiq
+  - TEST_REPO=manageiq#20018
+  - TEST_REPO=manageiq-api
+  - TEST_REPO=manageiq-providers-nuage
```
Comment (agrare) [inline .travis.yml]: Nuage is a bad one to use, it fails because we don't call `bin/qpid_install.sh` in cross-repo (working on it)

Comment (himdel) [reply .travis.yml]: ah, updating to vmware then

Comment (Fryguy) [reply .travis.yml]: hilariously that would be fixed with a before_install in nuage :)

Comment (agrare) [reply .travis.yml]: Haha yup exactly

**Pull request closed without merging**